### ISSUE TITLE
consider moving to displaying milliBitcoins over Bitcoins for greater ease of use

### ISSUE THREAD

**Issue by shea256** [blockstack/blockstack-browser#613]
I'm not sold on this idea but it has come up before and I think it might be good to consider doing this.

**Comment by larrysalibra** [comment]
I personally prefer bits - probably since I'm just used to using them in bread wallet, etc

**Comment by plus-x** [comment]
Yeah, it's extremely bad UX to expect the user to mouse over the BTC total in the wallet to wait for the tiny little popup tooltip to appear, showing the correct amount.

Why not just use that correct amount, already showing in the hover-tooltip, on the main display?

**Comment by guylepage3** [comment]
>I personally prefer bits

+1 on displaying bits.
